src/Ads/AssetSuggestionsService.php

```diff
@@ -20,20 +22,88 @@
  */
 class AssetSuggestionsService implements Service {
 
+	/**
+	 * WP Proxy
+	 *
+	 * @var WP
+	 */
+	protected WP $wp;
+
+	/**
+	 * WC Proxy
+	 *
+	 * @var WC
+	 */
+	protected WC $wc;
+
+	/**
+	 * Image utilities.
+	 *
+	 * @var ImageUtility
+	 */
+	protected ImageUtility $image_utility;
+
+	/**
+	 * Image requirements.
+	 *
+	 * @var array
+	 */
+	protected array $image_requirements = [
+		self::MARKETING_IMAGE_KEY        => self::MARKETING_IMAGE_SIZES,
+		self::SQUARE_MARKETING_IMAGE_KEY => self::MARKETING_SQUARE_IMAGE_SIZES,
+		self::LOGO_IMAGE_KEY             => self::LOGO_IMAGE_SIZES,
+
+	];
```

Comment:
By moving this here wouldn't we also want to make it a `const` or are we going to filter this at some point to allow additional sizes?

Also I understand the need for needing a unique prefixed key name where we want to have a const, but do we also need a const for each of the image size arrays? Probably just a nitpick but I have the feeling I need to scroll back and forth to figure out what all the images sizes are. So defining it all in one seems more readable:
```suggestion
	protected const $image_requirements = [
		self::MARKETING_IMAGE_KEY        => [
			'minimum'     => [ 600, 314 ],
			'recommended' => [ 1200, 628 ],
		],
		self::SQUARE_MARKETING_IMAGE_KEY => [
			'minimum'     => [ 300, 300 ],
			'recommended' => [ 1200, 1200 ],
		],
		self::LOGO_IMAGE_KEY             => [
			'minimum'     => [ 128, 128 ],
			'recommended' => [ 1200, 1200 ],
		],
	];
```

Reply:
> By moving this here wouldn't we also want to make it a const or are we going to filter this at some point to allow additional sizes?

I thought to leave it more flexible, thinking that maybe in the future, we could apply a filter so the user can update the sizes by themselves. 

> Also I understand the need for needing a unique prefixed key name where we want to have a const, but do we also need a const for each of the image size arrays?

Yes, I think it makes it easier to read all the sizes. I have updated the code: [f3a1a00](https://github.com/woocommerce/google-listings-and-ads/pull/1804/commits/f3a1a001127604f38fe52788e186c2ea3d65ab2f)

Reply:
I updated the variable `$image_requirements` to be a constant, if in the future we would like to apply a filter o to make it more dynamic we can always change it back.